rootfs/Dockerfile

```diff
@@ -8,8 +8,25 @@ RUN addgroup --quiet --gid 2000 slug && \
 RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
     apt-get update && \
     apt-get install -y md5deep && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
```

Comment:
@kmala ah right but that is when we build slugs, not when we build the image for the slugbuilder itself, right?